packages/flutter_list_bloc/example/lib/utils/continuous_offset_filter.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:list_bloc/list_bloc.dart';
+
+class ContinuousOffsetFilter extends OffsetLimitFilter {
+  ContinuousOffsetFilter({this.limit = 1000, this.offset = 0});
+
+  @override
+  final int limit;
+  @override
+  final int offset;
+
+  @override
+  copyWithOffset(int offset) => ContinuousOffsetFilter(
+        limit: limit,
+        offset: offset,
+      );
+
+  int get page => offset ~/ OffsetLimitFilter.kPageSize;
```

Comment:
@maksganzol Why not use the `limit` to calculate the page instead of `kPageSize`?

Reply:
@adar2378 Hm, I can't understand how limit can help to get know current page... 
Limit says about total items quantity on the resource, doesn' it?

Reply:
limit indicates how many items shall be fetched maximum. Same as what you are doing with kPageSize.

Reply:
But page size indicated how many items on one page, not items at all

Reply:
If resolve this then rest of the code looks good to me for this PR :)

Reply:
good job!

Reply:
Yes, now i got it, will fix